CHANGELOG.md

```diff
@@ -250,8 +250,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - New `SoundContainer` INI and Lua (R/W) property `PitchVariation`, which can be used to randomize the pitch of the sounds being played.
 
+- New `AHuman` and `ACrab` INI and Lua (R/W) property `JetReplenishRate`, which determines how fast jump time (i.e. jetpack fuel) is replenished during downtime.
```

Comment:
In INI this property is named `JumpReplenishRate`, so this changelog entry has to be tweaked I guess (or it has to be renamed in ini or lua so they're the same).

Reply:
Agreed, though I'm also okay holding off on this til we make jetpack a separate class (potentially extending some ChestModule or whatever class, so we can support different movement things) and have those property definitions move into it instead, that way we won't have to worry about the naming issue lol.